### PR TITLE
#25 (1 day) FRONT - Option to download work certification

### PR DESCRIPTION
**Pull request by yayobyte**
@hachesilva, Please code review.

### CODE REVIEW

File: src/js/app/modules/users/controllers/listUsers.controller.js [hunk 1]
```diff
@@ -58,5 +58,29 @@
                 vm.postStatus.postedForm = true;
             });
         };
+
+        vm.getCertification = function (userIdentification, typeOfCertification){
+            console.log(userIdentification, typeOfCertification);
+            userIdentification = (userIdentification != undefined) ? userIdentification : '0';
```
Comment (hachesilva) [inline src/js/app/modules/users/controllers/listUsers.controller.js]: Why the value coercion to `0` when `userIdentification` is `undefined`? shouldn't you just return to stop right here?

File: src/js/app/modules/users/controllers/listUsers.controller.js [hunk 2]
```diff
@@ -58,5 +58,29 @@
                 vm.postStatus.postedForm = true;
             });
         };
+
+        vm.getCertification = function (userIdentification, typeOfCertification){
+            console.log(userIdentification, typeOfCertification);
```
Comment (hachesilva) [inline src/js/app/modules/users/controllers/listUsers.controller.js]: console.log passed needs to be removed

File: src/js/app/modules/users/controllers/listUsers.controller.js [hunk 3]
```diff
@@ -58,5 +58,29 @@
...
+            GenerateCertificationFactory.query({action : certificateAction, id: userIdentification}, vm.getSuccess, vm.getError);
+        };
+
+        vm.getSuccess = function (response){
+            vm.postStatus  =  response[0];
+            var fileId = vm.postStatus.tempFileId;
+            window.open(globalConfig.apiEndpoint + globalConfig.apiRoutes.downloadCertification.replace(':file' , fileId));
+        };
+        vm.downloadSuccess = function (response){
+            console.log('Que cuca');
```
Comment (hachesilva) [inline src/js/app/modules/users/controllers/listUsers.controller.js]: Awesome console log message but needs to be gone :(

**Comment by hachesilva** [comment]
After addressing my comments, this should be ready to go.

File: src/js/app/modules/users/controllers/listUsers.controller.js (hunk 3 above)
Comment (yayobyte) [inline src/js/app/modules/users/controllers/listUsers.controller.js]: Lol. That's why I preffer to code review

**Pull request merged**